src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/AbstractPtvInTest.java

```diff
@@ -0,0 +1,220 @@
+    result.setOpeningHour(openingHour);
+    result.setServiceHourType(serviceHourType);
+    result.setValidForNow(validForNow);
+    result.setValidFrom(validFrom);
+    result.setValidTo(validTo);
+    return result;
+  }
+
+  protected ServiceHour createServiceHour(Boolean isClosed, List<DailyOpeningTime> openingHour, String serviceHourType, Boolean validForNow, OffsetDateTime validFrom, OffsetDateTime validTo, List<LocalizedValue> additionalInformation) {
+    ServiceHour result = new ServiceHour();
+    result.setAdditionalInformation(additionalInformation);
+    result.setIsClosed(isClosed);
+    result.setOpeningHour(openingHour);
+    result.setServiceHourType(serviceHourType);
+    result.setValidForNow(validForNow);
+    result.setValidFrom(validFrom);
+    result.setValidTo(validTo);
+    return result;
+  }
+
+  protected List<VmOpenApiWebPageWithOrderNumber> createPtvInWebPages(String language, String url, String value) {
+    VmOpenApiWebPageWithOrderNumber result = new VmOpenApiWebPageWithOrderNumber();
+    result.setLanguage(language);
+    result.setOrderNumber("0");
+    result.setUrl(url);
+    result.setValue(value);
+    return Arrays.asList(result);
+  }
+
+  protected List<V7VmOpenApiAddressWithMovingIn> createPtvInAddressAbroad(List<VmOpenApiLanguageItem> locationAbroad) {
+    return createPtvInAddress("Location", PtvAddressSubtype.ABROAD.getPtvValue(), null, null, null, Collections.emptyList(), locationAbroad);
+  }
+
+  protected List<V7VmOpenApiAddressWithMovingIn> createPtvInAddress(String type, String subType, String country, VmOpenApiAddressStreetWithCoordinatesIn streetAddress, VmOpenApiAddressPostOfficeBoxIn postOfficeBoxAddress, List<VmOpenApiAddressStreetWithCoordinatesIn> multipointLocation, List<VmOpenApiLanguageItem> locationAbroad) {
+    V7VmOpenApiAddressWithMovingIn result = new V7VmOpenApiAddressWithMovingIn();
+    result.setCountry(country);
+    result.setLocationAbroad(locationAbroad);
+    result.setMultipointLocation(multipointLocation);
+    result.setPostOfficeBoxAddress(postOfficeBoxAddress);
+    result.setStreetAddress(streetAddress);
+    result.setSubType(subType);
+    result.setType(type);
+    return Arrays.asList(result);
+  }
+  
+  protected List<Address> createAddresssesAbroad(List<LocalizedValue> locationAbroad) {
+    return createAddressses("Location", PtvAddressSubtype.ABROAD.getPtvValue(), null, null, null, null, null, locationAbroad, null, null, null, null, null, null, null, null);
+  }
+
+  protected List<Address> createAddressses(String type, String subtype, String country, List<LocalizedValue> streetAddress, String streetNumber, List<LocalizedValue> postOffice, List<Address> multipointLocation, List<LocalizedValue> locationAbroad, String latitude, String longitude, Municipality municipality, String postalCode, List<LocalizedValue> postOfficeBox, String coordinateState, Coordinates coordinates, List<LocalizedValue> additionalInformations) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 16 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)